src/main/resources/js/lgtm-addon.js

```diff
@@ -304,6 +210,43 @@ function updateConfig() {
 				fadeout : true
 			});
 			return;
+		} else if (jqXHR.getResponseHeader("Error") === "project") {
+			// This should not happen
+			AJS.messages.error("#message-context", {
+				title : 'The project ' + AJS.$(projectFieldId).select2('data').text
+						+ ' does not exist.',
+				closeable : true,
+				fadeout : true
+			});
+			return;
+		} else if (jqXHR.getResponseHeader("Error") === "issueType") {
+			AJS.messages.error("#message-context", {
+				title : 'The LGTM alert issue type doe not exist.',
```

Comment:
The error messages are temporal, could you help me find some better messages?

Reply:
`doe` -> `does` 

Perhaps `The "LGTM alert" issue type could not be found.`